packages/server/src/services/agent.service.ts

```
@@ -33,14 +33,32 @@ export class AgentService implements IAgentService {
   async handleUserRequest(
     agent: AgentSystem,
     userRequest: MessageRequest
+  ): Promise<AgentExecutionResponse>;
+  async handleUserRequest(
+    agent: any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)